packages/mds-agency-cache/index.ts

```diff
@@ -453,6 +387,7 @@ async function readDevicesStatus(query: {
     Object.assign(deviceStatusMap[item.device_id], item)
   })
   const values = Object.values(deviceStatusMap)
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I would rather leave these sorts of warnings in place to identify opportunities to tighten up the typings.